sci-misc/boinc/boinc-7.6.31.ebuild

```diff
@@ -0,0 +1,127 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```

Comment:
@jlec we are currently in the process of co-ordinating with Ryan in order to have wxwidgets.eclass be `EAPI=6` compliant.